chains/addresses.yaml

```diff
@@ -944,6 +944,21 @@ cyber:
   testRecipient: "0x2c61Cda929e4e2174cb10cd8e2724A9ceaD62E67"
   timelockController: "0x0000000000000000000000000000000000000000"
   validatorAnnounce: "0x062200d92df6bb7ba89ce4d6800110450f94784e"
+deepbrainchaintestnet:
```

Comment:
should we expect this change?

Reply:
yup - because this file wasn't autoupdated with the community PR